components/combobox/combobox.jsx

```diff
@@ -134,6 +142,17 @@ const propTypes = {
 	 * Forces the dropdown to be open or closed. See controlled/uncontrolled callback/prop pattern for more on suggested use view [Concepts and Best Practices](https://github.com/salesforce-ux/design-system-react/blob/master/CONTRIBUTING.md#concepts-and-best-practices) _Tested with snapshot testing._
 	 */
 	isOpen: PropTypes.bool,
+	/**
+	 * Sets the dialog width to the width of one of the following:
+	 * `target`: (Menus attached to `input` typically follow this UX pattern),
+	 *  `menu`: You may set a menuMaxWidth if using this value. If not, width will be set to width of largest menu item.
+	 *  'none'
+	 */
+	inheritWidthOf: PropTypes.oneOf([
```

Comment:
Allowing dialog width to be width of target, menu or none. This is a refactoring of `inheritTargetWidth`, which wasn't exposed.